Makefile.defs

```diff
@@ -111,16 +114,14 @@ ifneq ($(shell command -v go-junit-report),)
 			-out "$(JUNIT_PATH)") \
 		>($(GO) run $(ROOT_DIR)/tools/testowners \
 			--code-owners=$(CODEOWNERS_PATH)) \
-		>(tparse $(GOTEST_FORMATTER_FLAGS)) \
-		>$(GO_TEST_OUT) \
-		| cat
+		/dev/stdout \
+		| tparse $(GOTEST_FORMATTER_FLAGS)
 else
 	GOTEST_FORMATTER = tee \
 		>($(GO) run $(ROOT_DIR)/tools/testowners \
 			--code-owners=$(CODEOWNERS_PATH)) \
-		>(tparse $(GOTEST_FORMATTER_FLAGS)) \
-		>$(GO_TEST_OUT) \
-		| cat
+		/dev/stdout \
+		| tparse $(GOTEST_FORMATTER_FLAGS)
```

Comment:
I think this _looks_ like a cleanup but if the `testowners`, `go-junit-report` or other commands emit any stdout, this could cause incorrect parsing into `tparse` because `tparse` is only expecting json formatted output from `go test -json`. 

If the goal of this PR is to add `-follow` to the flags in specific circumstances, we have a lower likelihood to break something if we avoid making these extra changes here.